testing/unittests/jlibtests.cpp

```diff
@@ -2686,6 +2686,9 @@ class JlibCompressionTestsStress : public CppUnit::TestFixture
             {
                 compressed.clear();
                 ICompressHandler &handler = iter->query();
+                //Ignore unusual compressors with no expanders...
+                if (strieq(handler.queryType(), "randrow"))
```

Comment:
randrow does not support and expander - so it fails later on.

Reply:
Is there any test or plan to implement one for randrow?

Reply:
no.